frontend/model/contracts/group.js

```diff
@@ -163,6 +163,42 @@ DefineContract({
       }
       return pendingMembers
     },
+    groupMembersSorted (state, getters) {
```

Comment:
Note: Advantages of creating this getter:

- It can be reused by across components (ex: GroupMembers and GroupMembersAllModal), making sure the displayed data is exactly the same in both views.
- The sort is calculated only once during the user visit, even if they visit the dashboard or members search multiple times during their session (major advantage compared to a component property that is recalculated in each component mount)